src/utils/init/utils.ts

```diff
@@ -1,53 +1,57 @@
 import { writeFile } from 'fs/promises'
 import path from 'path'
 
+import { NetlifyConfig } from '@netlify/build'
+import { Settings } from '@netlify/build-info'
 import cleanDeep from 'clean-deep'
 import inquirer from 'inquirer'
 
+import BaseCommand from '../../commands/base-command.js'
 import { fileExistsAsync } from '../../lib/fs.js'
 import { normalizeBackslash } from '../../lib/path.js'
 import { detectBuildSettings } from '../build-info.js'
 import { chalk, error as failAndExit, log, warn } from '../command-helpers.js'
 
 import { getRecommendPlugins, getUIPlugins } from './plugins.js'
 
-// these plugins represent runtimes that are
-// expected to be "automatically" installed. Even though
-// they can be installed on package/toml, we always
-// want them installed in the site settings. When installed
-// there our build will automatically install the latest without
-// user management of the versioning.
-const pluginsToAlwaysInstall = new Set(['@netlify/plugin-nextjs'])
+const formatTitle = (title: string) => chalk.cyan(title)
 
 /**
  * Retrieve a list of plugins to auto install
- * @param {string[]=} pluginsInstalled
- * @param {string[]=} pluginsRecommended
+ * @param pluginsToAlwaysInstall these plugins represent runtimes that are
+ * expected to be "automatically" installed. Even though
+ * they can be installed on package/toml, we always
+ * want them installed in the site settings. When installed
+ * there our build will automatically install the latest without
+ * user management of the versioning.
+ * @param pluginsInstalled
+ * @param pluginsRecommended
  * @returns
  */
-export const getPluginsToAutoInstall = (pluginsInstalled = [], pluginsRecommended = []) =>
-  pluginsRecommended.reduce(
+export const getPluginsToAutoInstall = (
+  command: BaseCommand,
```

Comment:
I don't see this parameter used anywhere - am I missing something?

Reply:
Nope it is a prerequisite for the follow up PR just forgot to remove it from this one. In the follow up I need to access the feature flags from the base command.